### DISCUSSION TITLE
与“显示Greasyfork用户注册时间，识别恶意评论”油猴脚本冲突，显示重复文本

### DISCUSSION THREAD

**Discussion by 0verleaves** [F9y4ng/GreasyFork-Scripts#311] [问答专区(Question / Answer)] [answered]
浏览器：Chrome121
脚本管理器：篡改猴5.0.1
脚本版本：2024.02.03.1
出现渲染问题的网站：greasyfork.org，sleazyfork.org
截图：
![屏幕截图 2024-02-19 220449](https://github.com/F9y4ng/GreasyFork-Scripts/assets/54617500/7db18eb6-0865-4df3-bba0-398052d0143c)
![屏幕截图 2024-02-19 220514](https://github.com/F9y4ng/GreasyFork-Scripts/assets/54617500/c3bf5e02-cdf3-4778-b18e-55fcf37acf6f)

**Comment by 0verleaves** [comment]
油猴脚本链接：[显示Greasyfork用户注册时间，识别恶意评论](https://greasyfork.org/zh-CN/scripts/450603-%E6%98%BE%E7%A4%BAgreasyfork%E7%94%A8%E6%88%B7%E6%B3%A8%E5%86%8C%E6%97%B6%E9%97%B4-%E8%AF%86%E5%88%AB%E6%81%B6%E6%84%8F%E8%AF%84%E8%AE%BA)

**Comment by F9y4ng** [comment, marked as answer]
这……你应该与那个脚本的作者联系吧？新增的文字不是字体渲染脚本产生的。

对于脚本冲突这种问题，除非是超大用户群体使用的脚本或是扩展，否则基本都是二选一的处理结果。

## 临时的处理办法
大概看了一下，你打开字体渲染设置界面，取消勾选**粗体修正**后，保存为站点独享数据。
但要根本上解决问题，建议你还是与该脚本作者联系修正此问题。

**Reply by 0verleaves** [reply]
好的，成功解决了，谢谢大佬
